examples/fccReconstruction.xml

```diff
@@ -839,7 +839,7 @@
     <!-- FIXME:Check if we really want to keep those -->
     <parameter name="KeepDaughtersPDG" type="IntVec">22 111 310 13 211 321 3120 </parameter>
     <!--Name of the MCParticle input collection-->
-    <parameter name="MCParticleCollection" type="string" lcioInType="MCParticle"> MCParticle </parameter>
+    <parameter name="MCParticleCollection" type="string" lcioInType="MCParticle"> MCPhysicsParticles </parameter>
```

Comment:
MCPhysicsParticles only exists if you run with the overlay processor, you have to add an empty overlay run
like this
https://github.com/iLCSoft/CLICPerformance/blob/6605c09bfd49c7af50f1971469cc420a9ab6192e/examples/clicReconstruction.xml#L16-L18